designs/Commands/clear-package.md

```diff
@@ -0,0 +1,92 @@
+# Clearing an specific package from NuGet cache
+
+* Status: **Incubating**
+* Author(s): [Fernando Aguilar](https://github.com/dominofire)
+
+The purpose is to add a feature to `nuget locals` command to delete a package from NuGet cache folder
+
+
+## Problem Background
+
+There's some interest on having features to delete an specific package instead of clearing the whole cache.
+
+- https://github.com/NuGet/Home/issues/5713 : Comments on deleting specific packages
+- https://github.com/NuGet/Home/issues/4980 : Comments on cache handling polices
+- https://github.com/chgill-MSFT/NuGetCleaner : A clean-up tool relying on last-acesss file metadata in NTFS file system
+
+In PackageReference world, the global packages folders are shared across all restored projects. 
+
+There's no current API or command to delete an specific package.
```

Comment:
There's no API to do it in a single call, but it's absolutely possible to chain together a few API calls to get the global packages folder from config, create a path resolver for that global packages folder, find the directory for a package, and then delete it.

Given this is the "problem background" section, it makes it sound like this is a problem (and neither the goals or non-goals sections below say whether this is in-scope of this design or not). I don't remember seeing any customers asking for an easier API in the NuGet SDK, so I suggest removing the mention of API here.